src/pages/api/submit.ts

```diff
@@ -9,6 +9,15 @@ export default async function handler(
     res.status(405).end(`Method ${req.method} Not Allowed`)
   }
   const data = req.body
+  const requiredParameters = [
+    'imageInfo',
+    'proposalInfo',
+    'taoVoting',
+    'disputableVoting',
+  ]
+  if (!requiredParameters.every((item) => data.hasOwnProperty(item))) {
```

Comment:
There is still an error handling missing, concerning about the env variables. I receive a 200 request status even if GITHUB_AUTH, DB_CONNECTION, IMGUR_CLIENT_ID, GITHUB_OWNER and GITHUB_REPO is not set up